server/@types/licenceApiImport/index.d.ts

```diff
@@ -2,26 +2,278 @@
+      id?: number
+      /** The coded value for this standard */
+      code?: string
+      /** The sequence of this standard condition */
+      sequence?: number
+      /** The text of this standard condition */
+      text?: string
+    }
+    /** Response object for a created licence */
+    CreateLicenceResponse: {
+      /** Internal identifier for this licence generated within this service */
+      licenceId: number
+      /** Licence type code */
+      licenceType: 'AP' | 'AP_PSS' | 'PSS'
+      /** The status of this new licence after creation */
+      licenceStatus: 'IN_PROGRESS' | 'SUBMITTED' | 'ACTIVE' | 'REJECTED' | 'SUPERSEDED'
+    }
     /** Describes a test data object */
     TestData: {
       /** The key */
       key: string
       /** The value */
       value: string
     }
+    /** Describes an additional condition */
+    AdditionalCondition: {
+      /** The internal ID for this additional condition for this licence */
+      id: number
+      /** Coded value for the additional condition */
+      code?: string
+      /** Sequence of this additional condition within the additional conditions */
+      sequence?: number
+      /** The textual value for this additional condition */
+      text?: string
+      /** The list of data items entered for this additional condition */
+      data: components['schemas']['AdditionalConditionData'][]
+    }
+    /** Describes the data entered for an additional condition */
+    AdditionalConditionData: {
+      /** The internal ID of this data item, for this condition on this licence */
+      id: number
+      /** The sequence of this data item, for this condition on this licence */
+      sequence: number
+      /** The description of this data item for this condition on this licence */
+      description?: string
+      /** The format of this data item */
+      format?: string
+      /** The value of this data item */
+      value?: string
+    }
+    /** Describes a bespoke condition on a licence */
+    BespokeCondition: {
+      /** The internal ID for this condition on this licence */
+      id: number
+      /** The sequence of this bespoke condition on this licence */
+      sequence?: number
+      /** The text of this bespoke condition */
+      text?: string
+    }
+    /** Describes a licence document within this service */
+    Licence: {
+      /** Unique identifier for this licence within the service */
+      id: number
+      /** The licence type code */
+      typeCode: 'AP' | 'AP_PSS' | 'PSS' | 'AP' | 'PSS' | 'AP_PSS'
+      /** The version number used for standard and additional conditions */
+      version?: string
+      /** The current status code for this licence */
+      statusCode?:
+        | 'IN_PROGRESS'
+        | 'SUBMITTED'
+        | 'ACTIVE'
+        | 'REJECTED'
+        | 'SUPERSEDED'
+        | 'IN_PROGRESS'
+        | 'SUBMITTED'
+        | 'REJECTED'
+        | 'ACTIVE'
+        | 'SUPERSEDED'
```

Comment:
Are these in twice?

Reply:
Yep, the script generated them twice for some reason... fixed now